searx/settings.yml

```diff
@@ -908,4 +908,4 @@ doi_resolvers :
   doai.io  : 'https://doai.io/'
   sci-hub.tw : 'https://sci-hub.tw/'
 
-default_doi_resolver : 'oadoi.org'
+default_doi_resolver : 'oadoi.org'
```

Comment:
nitpicking: could you add the new line ? So settings.yml don't get involved in the PR.

Reply:
Done!